src/test/java/tests/GoodsInCartTest.java

```diff
@@ -1,37 +1,35 @@
 package tests;
 
 import bo.CartActions;
+import com.google.inject.Guice;
 import config.PropertyFile;
-import factory.DriverFactory;
-import org.testng.annotations.AfterMethod;
+import modules.CartPageModule;
+import modules.CartValidatorModule;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 import validators.CartValidator;
 
 import java.util.List;
 
-public class GoodsInCartTest {
+public class GoodsInCartTest extends BaseTest {
     private CartActions cartActions;
     private CartValidator validator;
 
     @BeforeMethod
-    public void setUp(){
-        cartActions = new CartActions();
-        validator = new CartValidator();
+    public void initialize() {
+        injector = Guice.createInjector(new CartPageModule());
```

Comment:
Why do you need this?
Cant you just method

all you need to write is
@Inject
private CartActions cartActions;

Reply:
without this method I have NullPointerExeptions